stk/utilities/utilities.py

```diff
@@ -1143,10 +1146,438 @@ def vector_theta(vector1, vector2):
     """
 
     numerator = np.dot(vector1, vector2)
-    denominator = (np.linalg.norm(vector1) *
-                   np.linalg.norm(vector2))
+    denominator = (np.linalg.norm(vector1) * np.linalg.norm(vector2))
     # This if statement prevents returns of NaN due to floating point
     # incurracy.
     if np.isclose(numerator, denominator, atol=1e-8):
         return 0.0
     return np.arccos(numerator/denominator)
+
+
+class XTBInvalidSolventError(Exception):
+    ...
+
+
+def is_valid_xtb_solvent(gfn_version, solvent):
+    '''Check if solvent is valid for the given GFN version.
```

Comment:
The first line of a docstring should be on the line below the quotes.

Also in stk three """ are used not '''